test/e2e/testsuites/azdiskschedulerextender_pod_scheduling_on_failover_multiple_pv_perf_tester.go

```diff
@@ -74,4 +78,11 @@ func (t *AzDiskSchedulerExtenderPodSchedulingOnFailoverMultiplePV) Run(client cl
 		}
 		wg.Wait()
 	}
+	//Check that AzVolumeAttachment resources were created correctly
+	time.Sleep(2 * time.Minute)
```

Comment:
Instead of a hardcoded sleep for two minutes, consider changing `VerifySuccessfulReplicaAzVolumeAttachments` to something like `GetReplicaAttachments` which returns an slice of `AzVolumeAttachment` CRIs for the pods. This would then be called in a polling loop until the desired number of attachments are found. It would be good to log the successful CRIs as they are found, and then log any unsuccessful CRIs with the `AzError` message if the loop times out. The polling interval could be something like 10s, and the timeout more like 10m for consistency with other wait patterns.